Authentication/Login/View/LoginConfiguration.swift

```diff
@@ -13,7 +13,7 @@ import Foundation
 */
 public protocol LoginControllerTransitionDelegate {
     
-    func loginControllerDidTapOnRegister(controller: LoginController)
+    func loginControllerDidTapOnSignup(controller: LoginController)
```

Comment:
I would change this completely when we add support for swift 3. I think a better name would be `onRegister` and `onSignup`. That is how I would it name it in any other language if it wasn't for UIKit weird conventions.